<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonPage</class>
 <widget class="QWidget" name="CommonPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="musicBox" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="cover">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>封面（图片）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonBox" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="commonText">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>本地音乐</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="playAll">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#playAll
{
    background-color:#e3e3e3;
    border-radius: 10px;
}

#playAll:hover
{
    background-color:#1ecd97;
}</string>
             </property>
             <property name="text">
              <string>播放全部</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addLocalBtn">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>440</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="musicDescribe" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>345</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>345</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>标题</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="singer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>歌手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="album">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>专辑</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="musicList">
     <property name="styleSheet">
      <string notr="true">#musicList::item:selected
{
	background-color:#ffffff;
	border:none;
}
#musicList::item:hover
{
	background-color:#ffffff;
}
#musicList::item
{
	background-color:rgb(247,249,252);
}
QScrollBar:vertical
{
	border:none;
	width:15px;
	background-color:rgb(247,249,252);
	margin:0px 0px 0px 0px;
}
QScrollBar::handle:vertical
{
	width:15px;
	border-radius:5px;
	background-color:rgb(225,228,233);
	min-height:20px;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
